remake.yml

```diff
@@ -31,15 +37,35 @@ targets:
     command: find_oldest_sites(states, parameter)
 
   # Pull site data
-  state_tasks:
+  state_combiners:
     command: do_state_tasks(
       oldest_active_sites,
       '1_fetch/src/get_site_data.R',
       '2_process/src/tally_site_obs.R',
-      '3_visualize/src/plot_site_data.R')
+      '3_visualize/src/plot_site_data.R',
+      '123_state_tasks.R')
```

Comment:
:+1: good addition here to make sure any changes to the combiners are recognized and a rebuild is triggered if needed.